src/lib/components/composites/paper-components/PaperInfo.svelte

```diff
@@ -34,11 +37,25 @@ for a project paper, the local / relative project paper id.
             {#if id}
                 <div class="text-default-sb-nc w-fit text-neutral-500">#{id}</div>
             {/if}
-            <h2 class="place-content-center truncate">{paper.title}</h2>
+            <h2 class="place-content-center truncate">
+                {#each highlightWords( { text: paper.title, query: safeSearchQuery, matchExactly: false }, ) as chunk (chunk.key)}
+                    {#if chunk.match}
+                        <span class="highlight">{chunk.text}</span>
+                    {:else}
+                        {chunk.text}
+                    {/if}
+                {/each}
+            </h2>
         </div>
         <div class="text-hint flex flex-row items-center truncate">
             {#if paper.authors.length > 0}
-                <span class="place-content-start truncate">{getNames(paper.authors)}</span>
+                {#each highlightWords( { text: getNames(paper.authors), query: safeSearchQuery, matchExactly: false }, ) as chunk (chunk.key)}
+                    {#if chunk.match}
+                        <span class="highlight">{chunk.text}</span>
+                    {:else}
+                        {chunk.text}
+                    {/if}
+                {/each}
```

Comment:
This still has to be in the span like before.
```suggestion
                <span class="place-content-start truncate">
                    {#each highlightWords( { text: getNames(paper.authors), query: safeSearchQuery, matchExactly: false }, ) as chunk (chunk.key)}
                        {#if chunk.match}
                            <span class="highlight">{chunk.text}</span>
                        {:else}
                            {chunk.text}
                        {/if}
                    {/each}
                </span>
```

Reply:
Good catch.